Eastern-Research-Group/mywaterway
# Features/417 grants section edits

## Related Issues:
* [HMW-417](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-417)

## Main Changes:
* Updated the GRTS "GetProjectsByHUC12" service endpoint to one that provides more detail in the `watershed_plans` field.
* Added a table to the accordion items in the Clean Water Act Section 319 Projects panel that lists a project's watershed plans and associated statuses.
* Changed the label of the "Total Funds" row to "Total EPA Funds", and added a "Total Budget" row (the former is some fraction of the latter).

## Steps To Test:
1. Go to http://localhost:3000/community/020802031302/restore.
2. Expand the accordion items in the _Clean Water Act Section 319 Projects_ panel. If the project has any watershed plans, a table should be displayed with the columns: **Watershed Plan** and **Watershed Plan Status**, with a table row for each plan. The plan name will be linked to a document, if available.
3. Expand the item "TMDL Implementation - Willis River and Slate River" (ID: 59128).
4. Verify that the plans display "Not Available" in the status column, since no statuses were supplied.
5. Verify that a plan displays "No Document Available" in the plan column if there is a status but no plan title. I could not find an example of this case, however, so code analysis may be the best option.
